e/eric.md

```diff
@@ -0,0 +1,13 @@
+# Eric Manana
```

Comment:
If this artist is called `Eric Manana` and not `Eric`, then we should update the filenames to be `eric-manana` and we fix the link in `e/index.md`

Reply:
I fixed it so please check it.

Thank you,